dstep/translator/Translator.d

```diff
@@ -199,6 +225,22 @@ private:
     }
 }
 
+string translateDefine(Cursor cursor) {
+
+    auto tokens = cursor.tokens();
+
+    if (tokens.length == 2) 
+    {
+        import std.format: format;
+        string result = format("enum %s = %s;", tokens[0].spelling, tokens[1].spelling);
+        return result;
+    }
+    else 
+    {
```

Comment:
I think `else` without braces when `if` has braces looks just ugly. I refactored in anyway.